pkg/fixtures/loader.go

```diff
@@ -25,7 +25,7 @@ func NewFixtureLoader(ctx context.Context, config cfg.Config, logger log.Logger)
 	logger = logger.WithChannel("fixture_loader")
 
 	settings := &fixtureLoaderSettings{}
-	config.UnmarshalKey("fixtures", settings)
+	config.UnmarshalKey("fixtureSets", settings)
```

Comment:
actually i personally do not like the renaming of the config key as the package itself is also named fixtures.
its personal preference i would say but it feels more consistent if the config key settings (root level) is the same as the package name.